ClassicAssist.Shared/UI/ViewModels/HotkeysTabViewModel.cs

```diff
@@ -56,7 +57,7 @@ public HotkeyEntry SelectedItem
             get => _selectedItem;
             set
             {
-                SetProperty( ref _selectedItem, value.IsCategory ? null : value );
+                SetProperty( ref _selectedItem, value == null || !value.IsCategory ? value : null );
```

Comment:
This doesn't really have to do with `MessageBox`, but I had an NPE crash if I click the "expand arrow" on a category entry: the `value` is null when clicking the expand icon vs the category text. Kinda odd...